packages/rocketchat-ui-flextab/client/tabs/userInfo.js

```diff
@@ -0,0 +1,526 @@
+/* globals RoomRoles, UserRoles*/
+import moment from 'moment';
+import toastr from 'toastr';
+
+Template.userInfo.helpers({
+	name() {
+		const user = Template.instance().user.get();
+		return user.name ? user.name : TAPi18n.__('Unnamed');
+	},
+
+	username() {
+		const user = Template.instance().user.get();
+		return user.username;
+	},
+
+	email() {
+		const user = Template.instance().user.get();
+		if (user.emails && user.emails[0] && user.emails[0].address) {
```

Comment:
Just as a note you may use `return user.emails && user.emails[0] && user.emails[0].address;`